rd-net/RdFramework.Reflection/ReflectionSerializerVerifier.cs

```diff
@@ -148,7 +148,10 @@ bool IsValidArray()
         if (!typeInfo.IsArray) return false;
         if (typeInfo.GetArrayRank() != 1) return false;
 
-        var arrayType = typeInfo.GetElementType().GetTypeInfo();
+        var elementType = typeInfo.GetElementType();
+        if (elementType == null) return false;
```

Comment:
Should never be `null` here, I'd prefer this code to fail in such a case (as it was before) instead of trying to handle the logic error.